src/tests/roc_audio/test_packetizer.cpp

```diff
@@ -69,7 +69,8 @@ class PacketChecker {
     }
 
     void read(packet::IReader& reader, size_t n_samples) {
-        packet::PacketPtr pp = reader.read();
+        packet::PacketPtr pp;
+        CHECK(reader.read(pp) == 0);
```

Comment:
In tests where we check error codes (for zero of for specific code), let's use LONGS_EQUAL instead of CHECK. It will print actual code if test fails, which makes it easier to debug failures.